ui/page/send/page.go

```diff
@@ -318,14 +338,28 @@ func (pg *Page) Handle() {
 		}
 	}
 
-	for pg.clearAllBtn.Clicked() {
+	for _, menu := range pg.moreItems {
+		if menu.button.Clicked() {
+			switch menu.id {
+			case UTXOPageID:
+				pg.ChangeFragment(NewUTXOPage(pg.Load))
+			default:
+				menu.action()
+			}
+		}
+	}
+
+	for pg.backdrop.Clicked() {
 		pg.moreOptionIsOpen = false
+	}
 
-		pg.sendDestination.clearAddressInput()
+}
 
-		pg.amount.clearAmount()
+func (pg *Page) mustIcon(ic *widget.Icon, err error) *widget.Icon {
+	if err != nil {
+		panic(err)
 	}
-
+	return ic
 }
```

Comment:
There are multiple instance of this already... Kindly remove and use on from the either the component/utils.go 

See comments above.